### PR TITLE
[FND 1656] Creating embedding with bedrock

### PR DESCRIPTION
**Pull request by sagar-thacker**

### CODE REVIEW

File: src/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import sys
+from pathlib import Path
+
+ROOT_PATH = Path(__file__).resolve().parent
+print(ROOT_PATH)
```
Comment (maxicecilia-esg) [inline src/__init__.py:5]: remove print

File: src/core/embeddings/bedrock.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+                input_body["dimensions"] = self.dims
+                input_body["normalize"] = self.normalize or False
+
+            input_body["inputText"] = text
+        else:
+            raise ValueError(f"Invalid model id: {self.model_id}")
+
+        body = json.dumps(input_body)
+        print(body)
```
Comment (maxicecilia-esg) [inline src/core/embeddings/bedrock.py:60]: lets replaces prints with `logger.debug`

File: src/create_embeddings.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import pandas as pd
```
Comment (maxicecilia-esg) [inline src/create_embeddings.py:1]: if you want to use the file as the wrapper around the functions, then lets change the name to something like `embeddings_service` or `embeddings_manager`, then the import `from src.embeddings_service import create_embeddings` looks way better.

you could rename `create_embeddings` -> `create`, but I can see both options working fine

File: src/create_embeddings.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    start_index = _find_first_nan_index(embedding_df)
+
+    if start_index and start_index != 0:
+        logger.info(
+            f"{start_index} rows already embedded. Resume embedding from {start_index + 1}..."
+        )
+
+    if start_index is None:
+        logger.info("All labels already embedded.")
+        return embedding_df
```
Comment (maxicecilia-esg) [inline src/create_embeddings.py:54]:
```suggestion
    if start_index and start_index != 0:
         logger.info(
            f"{start_index} rows already embedded. Resume embedding from {start_index + 1}..."
        )
    elif start_index is None:
        logger.info("All labels already embedded.")
        return embedding_df
```

this way you only check the second condition when needed

File: src/create_embeddings.py (hunk 3 above)
Comment (sagar-thacker) [reply src/create_embeddings.py:1]: I have kept `create_embeddings` because it's explicit and makes it clearer.

**Review by maxicecilia-esg — APPROVED** [review]

**Pull request merged**